dbschema/language.esdl

```diff
@@ -1,8 +1,5 @@
 module default {
-  type Language extending Resource, Project::ContextAware, Mixin::Pinnable, Mixin::Taggable {
-    required name: str;
```

Comment:
Why doesn't this still exist (but as `overloaded`) like it does on `Project`?

Reply:
Project names are exclusive. Language names are not...for reasons...probably bad ones but @sethmcknight has convinced me of it before.

Reply:
Ah, got it now!